qemu/deps/win_driver_install/win_driver_install.py

```diff
@@ -88,7 +88,7 @@ def get_inf_files(driver_path, driver_name):
     inf_files = []
     for root, dirs, files in os.walk(driver_path):
         files_path = map(lambda x: os.path.join(root, x), files)
-        inf_files += filter(lambda x: x.endswith(inf_name), files_path)
+        inf_files += filter(lambda x: x.lower().endswith(inf_name), files_path)

```

Comment:
@xutian 
Have updated the code with your comments.
Could you help review it again?

Thanks.